### DISCUSSION TITLE
Benchmark pretrained weights

### DISCUSSION THREAD

**Discussion by Alexhk01** [open-edge-platform/anomalib#1626] [Feature Requests]
### What is the motivation for this task?

My I ask why there are no pretrained weights  or custom trained weights configuration in benchmark pipeline?

### Describe the solution you'd like

For example, I haved trained an efficientAD model， how can I use benchmark to test my trained weights?

**Comment by samet-akcay** [comment]
@qwzhuanghkclr, I'm not sure if it is available at this moment. @ashwinvaidya17 might be able to help better here. 

We will be addressing the benchmarking pipeline in anomalib v1.1. Maybe we could label this as a feature request

**Comment by ashwinvaidya17** [comment]
Yeah, we are going to revamp the benchmarking stuff in anomalib v1.1. The current (main branch) benchmarking script uses the name of the model defined in the  benchmarking yaml file to fetch the config for that model. So all the parameters from the model's config file are loaded. If you want to use pretrained weights with benchmarking script then you need to set `resume_from_checkpoint` under trainer in `models/efficient_ad/config.yaml` . 
Other option would be to directly change this line https://github.com/openvinotoolkit/anomalib/blob/main/tools/benchmarking/benchmark.py#L97
